react/wbooks/src/app/components/NavBar/styles.module.scss

```diff
@@ -0,0 +1,26 @@
+@import '../../../scss/colors';
+
+.logout {
+  font-size: 16px;
+  font-weight: 700;
+  margin-left: auto;
+}
+
+.navbar {
+  border-top: 8px solid $cerulean;
+  box-shadow: 4px 4px 10px 0 $lighten-darken;
+  height: 100%;
+  max-height: 70px;
+  position: fixed;
+  top: 0;
```

Comment:
it's better to add one from Y axis and for X axis too. `left: 0;` because in some browsers or versions this could be badly rendered